lib/srv/desktop/rdp/rdpclient/src/rdpdr/mod.rs

```diff
@@ -2070,6 +2085,94 @@ impl FileBothDirectoryInformation {
     }
 }
 
+/// Base size of the FileFullDirectoryInformation, not accounting for variably sized file_name.
+/// Note that file_name's size should be calculated as if it were a Unicode string.
+/// 4 u32's (including FileAttributesFlags) + 6 i64's
+const FILE_FULL_DIRECTORY_INFORMATION_BASE_SIZE: u32 = (5 * 4) + (6 * 8); // 68
```

Comment:
4 u32s and 6 i64s would be `4 * 4 + 6 * 8 = 16 + 48 = 64` so this looks wrong to me.

Reply:
Changed the comment to what is should be:
```rust
/// 5 u32's (including FileAttributesFlags) + 6 i64's
```